### PR TITLE
Dropping fromRows for toRows

### PR DESCRIPTION
**Pull request by affo**
Related to #302.

`fromRows` is buggy, it needs to be split into `from` and `toRows`.

### CODE REVIEW

File: functions/transformations/transformations_test.go [hunk 1]
```diff
@@ -22,6 +22,7 @@ var skipTests = map[string]string{
 	"string_interp":             "string interpolation not working as expected in flux (https://github.com/influxdata/platform/issues/404)",
 	"to":                        "to functions are not supported in the testing framework (https://github.com/influxdata/flux/issues/77)",
 	"task_per_line":             "bug in group by caused by heterogeneous table schemas (https://github.com/influxdata/flux/issues/100)",
+	"fromRows":                  "buggy, dropping support (https://github.com/influxdata/flux/issues/302)",
```
Comment (nathanielc) [inline functions/transformations/transformations_test.go]: Can we fix this test as part of this change?

**Comment by nathanielc** [comment]
>  I think this is OK but toRows is not generally applicable except for results from influx storage. it will probably be a good fit once we have namespaces. should we keep for now?

I think this should inform what we name it. And yes when we have namespaces we will want to place it in an influx specific namespace.

We can pick a descriptive name for it now, then when we have namespaces we can rename it accordingly. For example `influxToRows` could become `influx.toRows`, or maybe we use `pivot`, `influxPivot` or `influx.pivot` when we have namespaces. Thoughts? Just brainstorming here, but we definitely need a better name.

**Comment by affo** [comment]
I opened an issue about a `select` statement and its relation with `toRows`. What do you guys think?

https://github.com/influxdata/flux/issues/312

Could `select` be an appropriate name? I thought about this because `toRows` performs a IFQL-style `SELECT *`.

**Comment by affo** [comment]
`select` implementation is postponed.

For a better name, here are my proposals:
 - `transpose` (very algebric)
 - `rows` (without "to" it seems less of an output)
 - `lines` (recalls the line protocol)
 - `pivotField` (describes what it does)

The name will be prefixed with `influx`.

@aanthony1243 can you comment on this?

**Comment by aanthony1243** [comment]
maybe...   `influxFieldsByRow` ?   the default is fields-by-column

**Comment by affo** [comment]
What do you think of `influxFieldsAsCols`?

It seems much clearer to me, don't you think?  
By default, field values are stored in rows 😄 

> For now, I left the .csv files for fromRows and skipped the tests, because fromRows leaves _start and _stop fields while toRows not.  
I saw that they are used in window, for instance. Should I keep them?  
Can I suppose that different fields for the same point have the same triple _start, _stop, _time?  
If so I could just use the 3 of them as rowKey in the pivoting operation.

Can you comment on this?

**Comment by aanthony1243** [comment]
`influxFieldsAsCols`  sounds good to me.  

if `_start` and `_stop` are dropped by pivot, it's because they are not part of the table key.  If you want them retained by the `pivot` call, then add a `group(by: ["_start", "stop", "_measurement", "_field"])`  before the pivot.  

I don't know if `fromRows`  will ever come back exactly as it was.  we can probably drop the tests.

**Comment by aanthony1243** [comment]
I think @nathanielc thumbs up is good enough, I dismissed his review this is OK to merge!

**Comment by affo** [comment]
Should I also update a SPEC, or should I file an issue as a reminder for the next release?

**Comment by aanthony1243** [comment]
@affo yes, submit a PR to update the spec please.

**Pull request merged**